plugins/processors/parser/README.md

```diff
@@ -1,21 +1,25 @@
 # Parser Processor Plugin
 
-This plugin parses defined fields containing the specified data format and
-creates new metrics based on the contents of the field.
+This plugin parses defined fields or tags containing the specified data format and
+creates new metrics based on the contents of the field or tag.
```

Comment:
Couldn't you use the converter processor to switch a tag to field, then use the parser plugin on the field?

Reply:
True, but the sample config already mentioned tags. Other processors als work on both tags and fields.

Reply:
@reimda ?